src/app/components/search/CustomFiltersManager.js

```diff
@@ -45,14 +47,15 @@ const messages = defineMessages({
   },
 });
 
-const CustomFiltersManager = ({
-  hide,
-  intl,
-  team,
-  operatorToggle,
-  onFilterChange,
-  query,
-}) => {
+const CustomFiltersManagerComponent = ({ data }) => {
+  const {
+    hide,
+    intl,
+    team,
+    operatorToggle,
+    onFilterChange,
+    query,
+  } = data;
```

Comment:
Please have each prop declared individually and explicitly, instead of collapsing all of them into a single `data` prop and please make sure that `CustomFiltersManager.propTypes` is consistent with the declared component props.